xml/System.Globalization/CultureInfo.xml

```diff
@@ -2911,7 +2911,7 @@ For a list of predefined culture names on Windows systems, see the **Language ta
       </ReturnValue>
       <Docs>
         <summary>Gets the culture name in the format *languagecode2*-*country/regioncode2*.</summary>
-        <value>The culture name in the format *languagecode2*-*country/regioncode2*. *languagecode2* is a lowercase two-letter code derived from ISO 639-1. *country/regioncode2* is derived from ISO 3166 and usually consists of two uppercase letters, or a BCP-47 language tag.</value>
+        <value>The culture name in the format *languagecode2*-*country/regioncode2*. *languagecode2* is a lowercase two-letter code derived from ISO 639-1. *country/regioncode2* is derived from ISO 3166 and usually consists of two uppercase letters, or a BCP-47 language tag. The invariant culture has an empty string.</value>
```

Comment:
```suggestion
        <value>The culture name in the format *languagecode2*-*country/regioncode2*, if the current <see cref="T:System.Globalization.CultureInfo" /> is culture-dependent; or an empty string, if it's an invariant culture. *languagecode2* is a lowercase two-letter code derived from ISO 639-1. *country/regioncode2* is derived from ISO 3166 and usually consists of two uppercase letters, or a BCP-47 language tag.</value>
```
Perhaps this wording would be better?

Reply:
This wording does flow better. Great suggestion!